Data protection for user profile data (#4492)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-exploit-guard/enable-controlled-folders-exploit-guard.md

>If controlled folder access is configured with Group Policy, PowerShell, or MDM CSPs, the state will change in the Windows Security app after a restart of the device.
>If the feature is set to **Audit mode** with any of those tools, the Windows Security app will show the state as **Off**.

> >If you are protecting user profile data, it is recommended that user profile should be on default Windows installation drive.

Reviewer: ```suggestion
>If you are protecting user profile data, it is recommended that the user profile should be on the default Windows installation drive.
```

> >If you are protecting user profile data, it is recommended that the user profile should be on the default Windows installation drive.

Reviewer 2: ```suggestion
>If you are protecting user profile data, we recommend that the user profile should be on the default Windows installation drive.
```
